suse_openstack_cloud

```diff
@@ -342,6 +342,8 @@ for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer \
     monasca-persister monasca-thresh trove designate sahara ironic ec2api; do
     find_and_pconf_files /etc/$svc -type f -regex \
         '.*\.\(co?nf\(ig\)?\|ini\|json\|filters\|y\(a\)?ml\)'
+    find_and_pconf_files /opt/stack/service/$svc -type f -regex \
+        '.*\.\(co?nf\(ig\)?\|ini\|json\|filters\|y\(a\)?ml\)'
```

Comment:
Unless I'm missing something you can avoid the duplication here:

    find_and_pconf_files /etc/$svc /opt/stack/service/$svc \
        -type f \
        -regex '.*\.\(co?nf\(ig\)?\|ini\|json\|filters\|y\(a\)?ml\)'